src/test/java/org/fundacion/jala/converter/core/FileStorageServiceTest.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Gustavo Zacarias Huanca Alconz
+ */
+package org.fundacion.jala.converter.core;
+
+import org.apache.commons.io.IOUtils;
+import org.fundacion.jala.converter.core.exceptions.FileStorageException;
+import org.junit.Test;
+import org.springframework.core.io.Resource;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.web.multipart.MultipartFile;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import static org.fundacion.jala.converter.core.FileStorageService.getOutputPath;
+import static org.fundacion.jala.converter.core.FileStorageService.getOutputPathWithoutFileName;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * This class is unit test for FileStorageService class
+ */
+public class FileStorageServiceTest {
+    FileStorageService fileStorageService = new FileStorageService();
+
+    @Test
+    public void uploadFile_MultipartFileRight_path() throws IOException, FileStorageException {
+        File file = new File("archive//archive.txt");
+        FileInputStream input = new FileInputStream(file);
+        MultipartFile multipartFile = new MockMultipartFile("archive", file.getName(), "text/archive",
+                IOUtils.toByteArray(input));
```

Comment:
Code lines can reach 120 characters, so you could move up this line

Reply:
it is not possible, if I could move up that line, it past 120 character